### ISSUE TITLE
log_logstash generates unsupported timestamp format

### ISSUE THREAD

**Issue by mguggi** [collectd/collectd#1132]
This plugin generates an unsupported timestamp format. According to the source code logstash supports the following formats: [Logstash](https://github.com/elastic/logstash/blob/master/spec/core/event_spec.rb#L268) 
```
"YYYY-MM-dd'T'HH:mm:ss.SSSZ",
"YYYY-MM-dd'T'HH:mm:ss.SSSSSSZ",
"YYYY-MM-dd'T'HH:mm:ss.SSS",
"YYYY-MM-dd'T'HH:mm:ss",
"YYYY-MM-dd'T'HH:mm:ssZ",
```
In case of this plugin the letter T as a delimiter is missing. This produces unnecessary warn messages in the log file.

```
{:timestamp=>"2015-07-12T13:27:14.717000+0000", :message=>"Error parsing @timestamp string,
setting current time to @timestamp, original in _@timestamp field", :value=>"\"2015-07-12 13:27:14Z\"",
:exception=>"invalid timestamp string \"2015-07-12 13:27:14Z\", error=java.lang.IllegalArgumentException: 
Invalid format: \"2015-07-12 13:27:14Z\" is malformed at \" 13:27:14Z\"", :level=>:warn}
```

Dependent issue is: log_logstash generates legacy logstash json format #974

**Comment by BCWilsonDotCom** [comment]
+1 I'm encounting this issue as well.

**Comment by mfournier** [comment]
Fixed in a46da00 (will be part of 5.5.1), thanks for reporting this !
